composites/Plugin/SnippetEditor/components/SnippetEditorFields.js

```diff
@@ -114,34 +117,55 @@ const SimulatedLabel = styled.div`
 	cursor: pointer;
 	font-size: 16px;
 	font-family: Arial, Roboto-Regular, HelveticaNeue, sans-serif;
+	margin-bottom: 5px;
+`;
+
+const TriggerReplacementVariableSuggestionsButton = Button.extend`
+	height: 33px;
+	border: 1px solid #dbdbdb;
+	box-shadow: none;
+	font-family: Arial, Roboto-Regular, HelveticaNeue, sans-serif;
```

Comment:
Why are we using this font stack? WP uses native fonts, we should remove this. There are other occurrences though, will open a separate issue.

Reply:
This was due to it having to match the old style for the color. I assume the same counts for the font stack. New issue sounds nice! On my todo is creating a issue to replace this with an IconButton.